packages/xrpl/src/utils/index.ts

```diff
@@ -194,6 +195,7 @@ export {
   deriveAddress,
   deriveXAddress,
   signPaymentChannelClaim,
+  verify,
```

Comment:
Is it not ambiguous to have a method just called `verify` in the utils?

Reply:
Ah, that is ambiguous. Good point.

Reply:
Renamed to `verifySignature`